curl/src/client.cc

```diff
@@ -101,6 +106,9 @@ namespace leatherman { namespace curl {
     {
         try {
             _temp_path = fs::path(file_path).parent_path() / fs::unique_path("temp_file_%%%%-%%%%-%%%%-%%%%");
+            #ifdef _WIN32
+            _fp = _fsopen(_temp_path.string().c_str(), "w+", _SH_DENYNO);
```

Comment:
This seems to result in an error such that the sha of the downloaded file does not match the expected sha. My guess is that the file is empty, but it could be an encoding issue? I think it may be worth looking in to temporarily keeping around the tempfile to take a look at what it looks like. Trace level debugging may help also.